docs/compliance.md

```
@@ -0,0 +1,265 @@
+### Regular usage pattern for tests
+
+As the format is to be evaluated from top to bottom, the typical sequence will be to have multiple blocks that all folow this pattern:
+
+* an opening `#`-comment-section
+* an initialising `=`-assignment-section
+* a number of variant `?`-template-sections that all are expected to produce the same result
+* a single `$`-result-section that captures the expected results for the previous templates
+* at least one, typically empty `#`-comment-section to wrap the result nicely 
+
+Example:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)